.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
   www_deploy:
     <<: *defaults
     docker:
-      - image: circleci/ruby:2.6.0-node
+      - image: circleci/ruby:2.7.1
```

Comment:
maybe even 2.7.3?
How about the new Circle CI builds? `cimg/ruby:2.7.3`
Might be worth a try as they will ensure they are regularly rebuilt with security patches

Reply:
The ios build is passing within circle Ci, I have tested it. Also upgraded ruby to 2.7.3 @steveyken please do have a look